index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>I'm Pretty Awesome</title>
+
+    <link rel="stylesheet" type=text/css href="assets/css/style.css">
+
+</head>
+<body>
+    
+        
+    <header id="header">
+    
+        <H1>Dylan Davidson</H1>
+
+       <div> <span id="links"> <a herf="https://rogue1037.github.io/Basic-Portfolio/">about</a>  portfolio contact</span> </div>
```

Comment:
I see you're using a `<div>` and a `<span>` tag here, you should definitely look up when to use each one and see if you can use them both together